ups/product_deps

```diff
@@ -248,8 +248,8 @@ fwdir	product_dir	scripts
 #
 ####################################
 product			version		qual	flags		<table_format=2>
-larpandoracontent	v04_08_01	-
-larreco			v09_25_02	-
+larpandoracontent	v04_09_00	-
+larreco			v09_25_03	-
```

Comment:
As above, I think we only want to include the actual source code changes in this PR.